packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -129,13 +129,25 @@ export class Governance {
   /**
    * Returns information about proposal votes.
    *
-   * @param {string} proposalId Proposal id
-   * @param {MasternodeType | string} [masternode=MasternodeType.MINE] masternode id or reserved words 'mine' to list votes for all owned accounts or 'all' to list all votes
-   * @param {number} [cycle=0] cycle: 0 (show current), cycle: N (show cycle N), cycle: -1 (show all)
+   * @param {ListGovProposalVotesOptions} options
+   * @param {string} options.proposalId Proposal id
+   * @param {MasternodeType | string} [options.masternode=MasternodeType.MINE] masternode id or reserved words 'mine' to list votes for all owned accounts or 'all' to list all votes
+   * @param {number} [options.cycle=-1] cycle: 0 (show current), cycle: N (show cycle N), cycle: -1 (show all)
+   * @param {ListGovProposalVotesPagination} [pagination]
+   * @param {number} [pagination.start] default is 0
+   * @param {boolean} [pagination.including_start] default = true
+   * @param {number} [pagination.limit] to limit number of records
    * @return {Promise<ListVotesResult[]>} Proposal vote information
    */
-  async listGovProposalVotes (proposalId: string, masternode: MasternodeType | string = MasternodeType.MINE, cycle: number = 0): Promise<ListVotesResult[]> {
-    return await this.client.call('listgovproposalvotes', [proposalId, masternode, cycle], 'number')
+  async listGovProposalVotes (
+    options: ListGovProposalVotesOptions,
+    pagination: ListGovProposalVotesPagination = {
```

Comment:
Correct me if i am wrong, from the code in `ain` https://github.com/DeFiCh/ain/blob/90c05a00d40381bf6da1b5db05a5e6301cd95a33/src/masternodes/rpc_proposals.cpp#L604

shouldn't `pagination` object be nested inside `options`?

Reply:
No, the [`pagination`](https://github.com/DeFiCh/ain/blob/90c05a00d40381bf6da1b5db05a5e6301cd95a33/src/masternodes/rpc_proposals.cpp#L524-L548) object is a top level argument, there is no `options` argument. 

In the RPC, all of the arguments can be passed as JSON object, which prevents having to define optional arguments. The [PR description](https://github.com/DeFiCh/ain/pull/1635) talks about this.

Reply:
ah okay got it, thanks for explaining!

Reply:
Misunderstood what you meant, we are using the JSON object here to pass all the arguments so the pagination object should be inside the options object. Fixed this in https://github.com/JellyfishSDK/jellyfish/pull/1908/commits/028f92c56498373a8a85f5b53fe607191a202711.

Thanks for bringing this up🫡